docs/index.rst

```diff
@@ -0,0 +1,57 @@
+.. F5 OpenStack LBaaSv1 documentation master file, created by
+   sphinx-quickstart on Mon Feb  8 14:03:09 2016.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+F5 OpenStack LBaaSv1
+================================================
+
+Contents:
+
+.. toctree::
+   :maxdepth: 2
+
+Introduction
+------------
+
+This site hosts the documentation for the F5 OpenStack OpenStack Neutron LBaaSv1 plugin and agent, used to control F5 BIG-IP devices in OpenStack. You can access the code and download release packages over on `GitHub <https://github.com/F5Networks/f5-openstack-lbaasv1>`__ 
```

Comment:
I'd like to see the actual linked text be F5Networks/f5-openstack-lbaasv1 instead of GitHub.